lib/pages/list.js

```diff
@@ -0,0 +1,207 @@
+    },
+
+    getInitialState: function() {
+        return {
+            pulls: {},
+            assignedPulls: {},
+            pullsState: unloaded,
+            assignedPullsState: unloaded,
+            error: null
+        };
+    },
+
+    loadPulls: function() {
+        var self = this;
+        self.props.controller.httpApi.getUser()
+            .done(function(userData) {
+                self.props.controller.httpApi.getOpenPullRequests(userData.body.login, "author")
+                .then(self._fillComments)
+                .then(self._loadMyPulls)
+                .catch(function(err) {
+                    console.error(JSON.stringify(err));
+                    self.setState({
+                        error: "Unexpected error loading pull requests"
+                    });
+                });
+
+                self.props.controller.httpApi.getOpenPullRequests(userData.body.login, "assignee")
+                .then(self._fillComments)
+                .then(self._loadAssignedPulls)
+                .catch(function(err) {
+                    console.error(JSON.stringify(err));
+                    self.setState({
+                        error: "Unexpected error loading pull requests"
+                    });
+                });
+            }, function(err) {
+            if (err.statusCode == 401) {
+                self.setState({error: "Error getting user details - try entering a valid access token"});
+            } else {
+                self.setState({error: "Unexpected error getting user details"});
+            }
+        });
+    },
+
+    _loadMyPulls: function(prs) {
+        var pulls = {};
+        pulls[unassignedKey] = [];
+        pulls[awaitingReviewKey] = [];
+        pulls[awaitingActionKey] = [];
+        pulls[reviewedKey] = [];
+        pulls[lgtmedKey] = [];
+        for (var i = 0; i < prs.length; ++i) {
+            var pr = prs[i];
+            if (!pr.getAssignee()) {
+                pulls[unassignedKey].push(pr);
+            } else if (pr.getComments().length <= 1 && pr.getActions().length === 0) { // 1 because creating the PR leaves a comment
+                pulls[awaitingReviewKey].push(pr);
+            } else if (pr.getActionsToDo().length > 0) {
+                pulls[awaitingActionKey].push(pr);
+            } else if (pr.isLGTM()) {
+                pulls[lgtmedKey].push(pr);
+            } else {
+                pulls[reviewedKey].push(pr);
+            }
+        }
+        this.setState({
+            pulls: pulls,
+            pullsState: loaded
+        });
+    },
+
+    _loadAssignedPulls: function(prs) {
+        var assignedPulls = {};
+        assignedPulls[assignedNeedsActionKey] = [];
+        assignedPulls[assignedOthersInvolvedKey] = [];
+        assignedPulls[assignedNoActionKey] = [];
+        for (var i = 0; i < prs.length; ++i) {
+            var pr = prs[i];
+            var me = pr.getAssignee();
+            if (pr.getOwner() === me) {
+                continue;
```

Comment:
Worth commenting why you're skipping this (it's not immediately clear because you can still be "assigned" to your own pull).